.release/buildspec_e2e.yml

```diff
@@ -31,6 +33,12 @@ batch:
           TEST_SUITE: customized-env
           APP_REGION: us-west-1
           APP_ACCOUNT: e2e/account/customized_env
+          TESTENV_REGION: us-west-1
+          TESTENV_ACCOUNT: e2e/account/customized_env
+          PRODENV_REGION: us-west-1
+          PRODENV_ACCOUNT: e2e/account/customized_env
+          SHAREDENV_REGION: us-west-1
```

Comment:
why do we need this shared env 🤔

Reply:
Because multi env has a "shared" environment 👀

Reply:
Do we need an env var for that? Or it is supposed to be using the same profile as whatever environment it is shared with?

Reply:
The environment's name is "shared": https://github.com/aws/copilot-cli/blob/83f09c7decb89560667f171e278aa57d72a05853/e2e/customized-env/customized_env_test.go#L102.

> Or it is supposed to be using the same profile as whatever environment it is shared with?

Did you mean create "shared" environment with the profile "test"?

Reply:
Discussed offline we'll keep as is so that the oncall don't need to look up which profile we use for an env by checking the e2e test file.